docker/cloudbuild.yaml

```diff
@@ -1,5 +1,5 @@
 # In this directory, run the following command to build this builder.
-# $ gcloud container builds submit . --config=cloudbuild.yaml
+# $ gcloud builds submit . --config=cloudbuild.yaml
```

Comment:
We can remove the `.` from this command too, and elsewhere.

Up to you.

Reply:
I'm going to do that in a separate PR -- want to get this through for the rename...